BasiliskII.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.haxxed.BasiliskII</id>
```

Comment:
Should be upstreamed as well

Reply:
That's the plan, yes. Just wanted to make sure it's acceptable before I merge it.

Reply:
No, it's fine to handle it later (either after merge or still during review).